algosdk/transaction.py

```diff
@@ -440,17 +440,17 @@ class KeyregTxn(Transaction):
         note (bytes)
         genesis_id (str)
         genesis_hash (str)
-        group(bytes)
-        votepk (str)
-        selkey (str)
+        group (bytes)
+        votekey (byte[32])
+        selkey (byte[32])
         votefst (int)
         votelst (int)
         votekd (int)
         type (str)
         lease (byte[32])
         rekey_to (str)
         nonpart (bool)
-        sprfkey (str)
+        sprfkey (byte[64])
```

Comment:
Expanding the input types to support bytes is a no-brainer, but I'm reluctant to change the attribute types as I view that as a breaking change.

The reality is this SDK uses base64 encoded strings to shuttle bytes around in a lot of places (e.g. public/private keys as well), and it's really in need of an upgrade to bytes across the board.